QuoteEngine/README.md

```diff
@@ -1,30 +1,177 @@
-Quick start, because I can't be arsed writing a full one ;)
+**Hello!**
 
-SQL setup
----------
 
-1. Create a user and database in your mysql server for the bot to use
-2. Populate the database using the .sql script
+Steps you need to do (or packages you should have) before downloading the script:
+1. Install all necessary packages for eggdrop to make use of mysql (also packages which you need to have are included).
+   <br>
+   For example for ubuntu 18.04 these would be the packages: tcl, tcl-dev, tcllib, tcl-tls, zlibc, zlib1g, tcl-trf, mysqltcl
+2. Install mysql-server and phpmyadmin (phpmyadmin is not really needed, but it helps to manage databases).
```

Comment:
Might be worth adding a note about being careful not to expose phpmyadmin to public access

Reply:
noted. added a few lines for that.